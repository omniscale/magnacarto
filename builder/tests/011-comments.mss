/* comment */
#class {
    line-width: 12;
}

/* comment with unicod√©  */

#class2 {
    line-width: 12;
}

// single line comment

@foo: 1 + 2;

// end of document comment