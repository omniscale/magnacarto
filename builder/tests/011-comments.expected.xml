<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="class2" filter-mode="first">
    <Rule>
      <LineSymbolizer stroke-width="12"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="class" srs="" status="off">
    <StyleName>class</StyleName>
  </Layer>
  <Layer name="class2" srs="" status="off">
    <StyleName>class2</StyleName>
  </Layer>
</Map>